about.html

```diff
@@ -0,0 +1,258 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+    />
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/js/bootstrap.bundle.min.js"></script>
+    <link rel="stylesheet" href="styles.css" />
+    <title>Web Development Course/About</title>
+  </head>
+  <body>
+    <div class="home-page-main-container">
+      <div class="side-nav open">
+        <ul class="side-nav-items-container">
+          <li class="side-nav-close-btn-container side-nav-size">
+            <img
+              src="images/close_mobile_menu.png"
+              alt="close button"
+              class="side-nav-close-btn"
+            />
+          </li>
+          <li class="side-nav-about-container side-nav-size">
+            <a href="about.html" class="sid-nav-about">About</a>
+          </li>
+          <li class="side-nav-program-container side-nav-size">
+            <a href="#" class="sid-nav-program">Program</a>
+          </li>
+          <li class="side-nav-join-container side-nav-size">
+            <a href="#" class="sid-nav-join">Join</a>
+          </li>
+          <li class="side-nav-sponser-container side-nav-size">
+            <a href="#" class="sid-nav-sponser">Sponser</a>
+          </li>
+          <li class="side-nav-news-container side-nav-size">
+            <a href="#" class="sid-nav-news">News</a>
+          </li>
+        </ul>
+      </div>
```

Comment:
![capstone-about-menu](https://user-images.githubusercontent.com/93039359/198582583-acc5a9e0-4932-40f0-8f20-ebe2ac292bcd.png)

- [ ] In the mobile version, Your nav menu of about page should have a link to your homepage instead of about page link as you can see in the image above :+1: